books/RayTracingInOneWeekend.html

```diff
@@ -547,13 +548,13 @@
 
         std::cout << "P3\n" << image_width << " " << image_height << "\n255\n";
 
-        for (int j = image_height-1; j >= 0; --j) {
-            std::clog << "\rScanlines remaining: " << j << ' ' << std::flush;
+        for (int j = 0; j < image_height; ++j) {
+            std::clog << "\rScanlines remaining: " << (image_height - j) << ' ' << std::flush;
             for (int i = 0; i < image_width; ++i) {
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
-                auto u = double(i) / (image_width-1);
-                auto v = double(j) / (image_height-1);
-                ray r(origin, lower_left_corner + u*horizontal + v*vertical - origin);
```

Comment:
If I understand correctly, flipping the count of `j` is the cause of the `(1-t)` in `(1-t)*vertical`

If you're going to stick with this updated convention, I'm going to ask you to justify the `(1-t)` in the text

Reply:
It looks like you do in a later commit?

Reply:
Yup, this is addressed in a subsequent pass. I just did code changes first.